src/routes/safe/components/Transactions/TxsTable/ExpandedTx/OwnersColumn/index.jsx

```diff
@@ -129,29 +170,66 @@ const OwnersColumn = ({
         showConfirmBtn={showConfirmBtn}
         showExecuteBtn={showExecuteBtn}
       />
+      {cancelTx && (
+        <>
+          <Block
+            className={cn(
+              classes.ownerListTitle,
+              (cancelThresholdReached || cancelTx.isExecuted) && classes.ownerListTitleCancelDone,
+            )}
+          >
+            <div className={classes.verticalLineProgressPending} />
+            <div className={classes.iconState}>
+              {cancelThresholdReached || cancelTx.isExecuted ? (
+                <Img src={CheckLargeFilledRedIcon} />
+              ) : (
+                <Img src={ConfirmLargeRedIcon} />
+              )}
+            </div>
+            {cancelTx.isExecuted
+              ? `Rejected [${cancelTx.confirmations.size}/${cancelTx.confirmations.size}]`
+              : `Rejected [${cancelTx.confirmations.size}/${threshold}]`}
+          </Block>
+          <OwnersList
+            userAddress={userAddress}
+            ownersWhoConfirmed={ownersWhoConfirmedCancel}
+            ownersUnconfirmed={ownersUnconfirmedCancel}
+            executor={cancelTx.executor}
+            thresholdReached={cancelThresholdReached}
+            onTxConfirm={onCancelTxConfirm}
+            onTxExecute={onCancelTxExecute}
+            showConfirmBtn={showConfirmCancelBtn}
+            showExecuteBtn={showExecuteCancelBtn}
+            isCancelTx
+          />
+        </>
+      )}
       <Block
         className={cn(
           classes.ownerListTitle,
           tx.isExecuted && classes.ownerListTitleDone,
+          cancelTx && cancelTx.isExecuted && classes.ownerListTitleCancelDone,
         )}
       >
         <div
           className={
-            thresholdReached || tx.isExecuted
+            tx.isExecuted
```

Comment:
can this be simplified? it's not a good practice to nest ternaries

Reply:
yes, it actually was to make it work but I completely forgot to refactor those nested ternaries.

Thanks for the catch